woocommerce/woocommerce
# Exclude coming soon patterns from block inserter

### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/48820.

Excludes all coming soon patterns and blocks from block inserter.



### How to test the changes in this Pull Request:

#### Prerequisite

1. `launch-your-store` feature flag is enabled
2. Your site must use FSE-supported theme like TT4

#### Test excluding blocks from block inserter

1. Go to `Pages > Add new page`
4. Click on add block icon
5. Filter for `coming soon`
6. Observe no blocks are found

<img width="1256" alt="image" src="https://github.com/woocommerce/woocommerce/assets/3747241/ac480b45-5859-46db-a2bc-68477863d249">

#### Regression test

1. Go to `WooCommerce > Settings > Site visibility`
2. Ensure `Coming soon` is selected and save
1. Go to `Appearance > Editor > Templates > Page: Coming soon`
3. Observe it shows the coming soon template

### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
